testah-junit/src/main/java/org/testah/framework/report/jira/JiraReporter.java

```diff
@@ -159,6 +160,37 @@ public ResponseDto updateRemoteLink(final String issue, final int id, final Remo
         return null;
     }
 
+    /**
+     * Check whether the Jira issue is closed.
+     *
+     * @param issue the Jira Id of the issue
+     * @return true if the Jira issue status is 'Closed'
+     */
+    public boolean isIssueClosed(final String issue) {
+        boolean status = false;
+        IssueStatus issueStatus = getStatus(issue);
+        if (issueStatus != null)
+        {
+            status = issueStatus.getFields().getStatus().getName().toLowerCase().equals("closed");
```

Comment:
possible null pointer issue chaining without checking, wrap in try catch to handle is does occur